libs/app-shell/src/lib/components/plugin-launcher/plugin-launcher.component.ts

```diff
@@ -101,61 +104,136 @@ export class PluginLauncherComponent implements OnChanges, AfterViewChecked {
             return;
         }
 
-        const _pluginModule = descriptor.modules.find(module => module.name === this.module);
-        const _remoteModule = await loadRemoteModule<DescriptorsModule>(descriptor, _pluginModule)
-            .catch(err => console.error(err));
-        if (!_remoteModule) {
+        const pluginModule = this.findPluginModule(descriptor);
```

Comment:
it could fail to find module too, plus convert it to `async` to allow using remote plugin descriptors eventually